models/jupiter/solana/jupiter_solana_perp_events_schema.yml

```diff
@@ -0,0 +1,44 @@
+version: 2
+
+models:
+  - name: jupiter_solana_perp_events
+    meta:
+      blockchain: solana
+      contributors: [ilemi]
+    config:
+      tags: ['solana','jupiter','perps']
+    description: >
+        solana perp events
```

Comment:
please add unique combo of columns test based on keys:
https://github.com/duneanalytics/spellbook/pull/5630/files#diff-639790f5883efe3bae264729381b3a22e1491e2d3a50e36667e8d60cc98c4a46R9